server/routes/csip-records/view.njk

```diff
@@ -88,7 +88,9 @@
     {% if decision %}
       {% include './components/decision.njk' %}
     {% endif %}
-    {% include './components/investigation.njk' %}
+    {% if not isUpdate or not decision %}
```

Comment:
Wont this try to show the investigation template if we're in update mode, even if an investigation doesn't exist?

Reply:
update-xxx controller redirects to csip-records page if it's not in the right status, so it wouldn't reach this bit.
we shouldn't set `referralTabSelected = false` if there isn't an investigation tab?